hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -286,6 +286,12 @@ private FlinkOptions() {
       .defaultValue(KeyGeneratorType.SIMPLE.name())
       .withDescription("Key generator type, that implements will extract the key out of incoming record");
 
+  public static final ConfigOption<Integer> INDEX_BOOTSTRAP_TASKS = ConfigOptions
+      .key("index_bootstrap.tasks")
+      .intType()
```

Comment:
Follow the name pattern for all the writer options: `write.index_bootstrap.tasks`.

Reply:
Ok, I design this name like BUCKET_ASSIGN_TASKS, I will modify this.